src/black/__init__.py

```diff
@@ -492,13 +492,14 @@ def validate_regex(
 @click.option(
     "--exclude",
     type=str,
-    default=DEFAULT_EXCLUDES,
     callback=validate_regex,
     help=(
         "A regular expression that matches files and directories that should be"
         " excluded on recursive searches. An empty value means no paths are excluded."
         " Use forward slashes for directories on all platforms (Windows, too)."
-        " Exclusions are calculated first, inclusions later."
+        " Exclusions are calculated first, inclusions later. [default: /(\\.direnv|"
+        "\\.eggs|\\.git|\\.hg|\\.mypy_cache|\\.nox|\\.tox|\\.venv|venv|\\.svn|_build|"
+        "buck-out|build|dist)/]"
```

Comment:
Unless I'm missing something, substituting the `DEFAULT_EXCLUDES` variable into the help text at runtime works just fine. If we ever update the `DEFAULT_EXCLUDES` we won't have go and update the help text (that's if we even remember to do so).
```suggestion
        f" Exclusions are calculated first, inclusions later. [default: {DEFAULT_EXCLUDES}]"
```